templates/header.html

```diff
@@ -3,6 +3,7 @@
 <meta name="keywords" content="HTML, CSS, XML, XHTML, JavaScript">
 <meta name="author" content="Felipe Torres">
 
+<a href="#skip_content" style="position:absolute; left:-10000px; top:auto; width:1px; height:1px; overflow:hidden;">Skip to content</a>
```

Comment:
styles should be in the stylesheet (e.g. `class="skip-to-content` then add .skip-to-content to the custom.css)